dev-infrastructure/configurations/svc-infra.tmpl.bicepparam

```diff
@@ -7,7 +7,7 @@ param serviceKeyVaultLocation = '{{ .serviceKeyVault.region }}'
 param serviceKeyVaultSoftDelete = {{ .serviceKeyVault.softDelete }}
 param serviceKeyVaultPrivate = {{ .serviceKeyVault.private }}
 
-param regionalDNSZoneName = '{{ .regionalDNSSubdomain}}.{{ .baseDnsZoneName }}'
+param svcParentZoneName = '{{ .svcParentZoneName }}'
```

Comment:
the full regional dns zone name is `frontend.{{ .regionalDNSSubdomain}}.{{ .svcParentZoneName }}`